influxdb3_server/src/http.rs

```diff
@@ -1784,11 +1782,7 @@ pub(crate) async fn route_request(
     let path = uri.path();
     // admin token creation should be allowed without authentication
     // and any endpoints that are disabled
-    if path == all_paths::API_V3_CONFIGURE_ADMIN_TOKEN
-        || paths_without_authz
-            .iter()
-            .any(|disabled_authz_path| *disabled_authz_path == path)
-    {
+    if path == all_paths::API_V3_CONFIGURE_ADMIN_TOKEN || paths_without_authz.contains(&path) {
```

Comment:
clippy suggested that `contains` was more efficient than `iter().any()`, which makes sense.